test/unit/clarity_ext/dilution/hamilton_tests.py

```diff
@@ -17,6 +20,9 @@ class HamiltonTests(unittest.TestCase):
     """
 
     def setUp(self):
+        self.mock = DilutionMock1()
```

Comment:
I agree that it makes sense to move this into another class when using this approach, but I'd like to show another approach. When that approach is used, it starts becoming questionable if it needs to be another class or not. It might be (making the test less readable, but more DRY) but it might also not be.